pom.xml

```diff
@@ -292,6 +292,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.mariadb.jdbc</groupId>
+            <artifactId>mariadb-java-client</artifactId>
+            <version>3.4.0</version>
```

Comment:
There should be no need for version here as the dependency is/should be defined at BOM lovel

Reply:
Oh, ok.
Is there anything else to do besides removing the version?